engine/src/main/java/org/pentaho/di/kitchen/KitchenCommandExecutor.java

```diff
@@ -381,4 +376,17 @@ protected void printRepositories( RepositoriesMeta repositoriesMeta ) {
   public void setKettleInit( Future<KettleException> kettleInit ) {
     this.kettleInit = kettleInit;
   }
+
+  private int getReturnCode() {
```

Comment:
Can I see unit tests for this new method?

Reply:
PR updated, adding unit tests.